app/models/OkcBlotter/booking.rb

```diff
@@ -0,0 +1,4 @@
+class OkcBlotter::Booking < ApplicationRecord
```

Comment:
This inherits from ApplicationRecord which means it's a model
Models in rails allow you to interact with a database table without using SQL and should contain information about their associations, validation, etc.